recipes-bsp/tegra-binaries/tegra-helper-scripts/make-sdcard.sh

```diff
@@ -141,6 +141,27 @@ make_partitions() {
     return 0
 }
 
+create_filesystems() {
+    local partnumber
```

Comment:
You should declare all of the variables that get set via the `eval "$pline"` as local here, as is done in the `write_partitions_to_{device,image}`  functions, and those functions should be updated to add the new `fstype` to their declarations (and it looks like they're missing at least `start_location` from theirs, too, oops).